site-src/references/policy-attachment.md

```diff
@@ -160,6 +173,7 @@ spec:
         includeProtocol: true
         includeQueryString: true
   targetRef:
+    type: hiearchical
```

Comment:
This feels like it's going to be the most commonly misspelled word in a k8s API :). If we can avoid this as an input altogether, I expect we'll avoid some frustration. 
```suggestion
    type: hierarchical
```

Reply:
Yeah, I don't like the name either, I've been noodling some others that I'll suggest in the meeting today.